examples/sglang_bench/bench_sharegpt.py

```
@@ -191,27 +466,122 @@ def run_benchmark(base_url: str, prompts: List[str], batch_size: int) -> list:
 
 
+def _shutdown_server(
+    process: subprocess.Popen,
+    peer_processes: List[subprocess.Popen],
+) -> None:
+    """Kill the SGLang head server and all peer workers."""
+    kill_process_tree(process.pid)
+    for p in peer_processes:
+        try:
+            peer_host = p.args[5]  # ssh cmd: ["ssh", "-o", OPT, "-o", OPT, HOST, remote_cmd]
+            subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)